spec/components/contextual_breadcrumbs_spec.rb

```diff
@@ -104,6 +104,14 @@ def set_live_taxons(content_item)
     assert_select "a", text: "Competition and Markets Authority cases"
   end
 
+  it "renders parent finder breadcrumb if content has a finder linked" do
+    content_item = example_document_for("guide", "guide-with-facet-groups")
+    render_component(content_item: content_item)
```

Comment:
We should add another test that sets `prioritise_taxon_breadcrumbs` to true.

Reply:
Done 👍